packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -245,6 +245,9 @@ fi
 %endif
 
 %changelog
+* Thu Aug 30 2018 Feng Xiong<fenxiong@amazon.com> - 1.20.2-1
```

Comment:
Please update this to have a space between your name and email. Can you also update the 1.20.1-1 entry in the same way?